testsuite/openshift/client.py

```diff
@@ -149,6 +149,22 @@ def create_tls_secret(self, name: str, certificate: Certificate, labels: Optiona
         with self.context:
             return oc.create(model, ["--save-config=true"])
 
+    def create_signing_secret(self, name: str, key: str, labels: Optional[Dict[str, str]] = None):
```

Comment:
Would it make sense to update the existing create_tls_secret method? e.g. ability to rename the keys?

Reply:
I would prefer to leave these secrets as they are for now. Because, in my opinion, they have a lot of differences, such as different types, key names, count of secret pairs.

Reply:
Would it work with both key and cert in the secret? If yes, then I think adding the option to rename the key and change the type would be enough and it wouldn't be that different